.github/workflows/linux_arm64.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         run: |
           bash ${OPENVINO_REPO}/install_build_dependencies.sh
           # default-jdk - Java API
-          apt install --assume-yes --no-install-recommends default-jdk
+          apt install --assume-yes --no-install-recommends default-jdk gcc-10-aarch64-linux-gnu g++-10-aarch64-linux-gnu          
```

Comment:
since we are compiling natively on arm64, we don't need to install cross toolchain. Just use gcc-10, etc

But we also need to set gcc-10 as default version to be used instead of  gcc-9. It can be solved via update alternatives priorities